build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
-    ext.kotlin_version = '1.3.10'
-    ext.arrow_version = '0.8.2-SNAPSHOT'
+    ext.kotlin_version = '1.3.21'
+    ext.arrow_version = '0.9.0-SNAPSHOT'
```

Comment:
0.9.0 is stable now :D

Reply:
I know, but this is the only way to call the dependencies. This was in the past an issue, which I also resolved on the webpage. Maybe, I have the wrong maven URL, but then we should change it also on the webpage. https://arrow-kt.io/docs/

Reply:
> What error do you see?

<img width="825" alt="Screenshot 2019-03-15 at 00 11 38" src="https://user-images.githubusercontent.com/46971368/54397534-1c580200-46b7-11e9-807a-f0efbfeee4b1.png">

should we rename it ? I am not an expert in gradle :)